.travis.yml

```diff
@@ -18,6 +18,17 @@ script:
 after_success:
   - node deploy-docs.js
   - if [[ "$TRAVIS_PULL_REQUEST" = "false" && ( "$TRAVIS_BRANCH" = "master" || "$TRAVIS_TAG" != "" ) ]]; then node_modules/.bin/grunt dists; fi
+before_deploy:
+  - npm run build-storybook
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GH_TOKEN
```

Comment:
@eonarheim do you recall if this has write privs? We also have $GH_DIST_TOKEN for writing to dist repo.

Reply:
Once it is merged, we will find out I guess 🤷‍♂️